### ISSUE TITLE
[Feature Request] Possible to reduce re-marks?

### ISSUE THREAD

**Issue by Skyl3lazer** [luigi311/JellyPlex-Watched#158]
I don't know the workflow for the script so I can't point to a way to do this, but it would be nice if there was a less spammy way to do the sync. Right now each user's entire watch history is marked in the other application every time it runs, which takes a lot of time for even a few users.  

Maybe a mode that only marks something as watched once? It's pretty rare for users to mark something un-watched again, so if something has been marked watched once I'd be happy with a mode that just wouldn't look at that item for that user again.

**Comment by luigi311** [comment]
I dont think this is something i will add. Right now it grabs the entire watch history, filters down to new objects and then only marks those as watched. If you want to avoid having it read through the entire watch history you are better off using something like https://github.com/arabcoders/watchstate which lets its listen on webhooks and sets the status on the other services as it runs without the need to fetch the entire history like mine does.
